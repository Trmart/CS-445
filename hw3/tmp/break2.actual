.   .   .   Child: 0  Const true [line: 7]
.   .   .   .   .   Child: 0  Const true [line: 8]
.   .   .   Child: 0  Id: i [line: 6]
.   .   Child: 0  Var: i of type int [line: 4]
.   .   Child: 1  Assign: <= [line: 6]
.   Child: 1  Compound [line: 3]
.   .   .   Child: 1  Compound [line: 7]
.   .   .   .   .   Child: 1  Compound [line: 8]
.   .   .   Child: 1  Const 0 [line: 6]
.   .   .   .   Child: 1  While [line: 8]
Func: ( returns type void [line: 2]
.   .   Sibling: 1  While [line: 7]
