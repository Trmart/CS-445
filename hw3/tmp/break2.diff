1,6c1,12
< ERROR(10): Symbol 'output' is not declared.
< ERROR(12): Symbol 'outnl' is not declared.
< ERROR(16): Symbol 'output' is not declared.
< ERROR(17): Symbol 'outnl' is not declared.
< Number of errors: 4
< Number of warnings: 0
---
> .   .   .   Child: 0  Const true [line: 7]
> .   .   .   .   .   Child: 0  Const true [line: 8]
> .   .   .   Child: 0  Id: i [line: 6]
> .   .   Child: 0  Var: i of type int [line: 4]
> .   .   Child: 1  Assign: <= [line: 6]
> .   Child: 1  Compound [line: 3]
> .   .   .   Child: 1  Compound [line: 7]
> .   .   .   .   .   Child: 1  Compound [line: 8]
> .   .   .   Child: 1  Const 0 [line: 6]
> .   .   .   .   Child: 1  While [line: 8]
> Func: ( returns type void [line: 2]
> .   .   Sibling: 1  While [line: 7]
