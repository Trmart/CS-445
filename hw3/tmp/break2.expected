ERROR(10): Symbol 'output' is not declared.
ERROR(12): Symbol 'outnl' is not declared.
ERROR(16): Symbol 'output' is not declared.
ERROR(17): Symbol 'outnl' is not declared.
Number of errors: 4
Number of warnings: 0
